CryptoTechReminderSystem/UseCase/RemindLateDevelopers.cs

```diff
@@ -7,25 +7,29 @@ public class RemindLateDevelopers
     {
         private IGetLateDevelopers _getLateDevelopers;
         private IRemindDeveloper _remindDeveloper;
+        private IClock _clock;
 
         public RemindLateDevelopers(IGetLateDevelopers getLateDevelopers, IRemindDeveloper remindDeveloper, IClock clock)
         {
             _getLateDevelopers = getLateDevelopers;
             _remindDeveloper = remindDeveloper;
+            _clock = clock;
         }
 
         public void Execute(RemindLateDevelopersRequest remindLateDevelopersRequest)
         {
-            var lateDevelopers = _getLateDevelopers.Execute();
-            foreach (var lateDeveloper in lateDevelopers.Developers)
+            if ((_clock.Now().ToUnixTimeSeconds() % 1800 == 0) && (_clock.Now().Hour < 14))
```

Comment:
I think we need to make sure that this can only be executed on a Friday as well.

Reply:
Resolved in https://github.com/chubberlisk/crypto_tech/pull/17/commits/e54643950a71d35df3a8635222a4fd2d83db66ee